### PR TITLE
add reviewer suggestions in PR #122

### PR DESCRIPTION
**Pull request by Daniel-VM**
## PR checklist

- [X] This comment contains a description of changes (with reason).
- [ ] If you've fixed a bug or added code that should be tested, add tests!
- [ ] If you've added a new tool - have you followed the pipeline conventions in the [contribution docs](https://github.com/nf-core/bacass/tree/master/.github/CONTRIBUTING.md)
- [ ] If necessary, also make a PR on the nf-core/bacass _branch_ on the [nf-core/test-datasets](https://github.com/nf-core/test-datasets) repository.
- [X] Make sure your code lints (`nf-core lint`).
- [X] Ensure the test suite passes (`nextflow run . -profile test,docker --outdir <OUTDIR>`).
- [ ] Usage Documentation in `docs/usage.md` is updated.
- [ ] Output Documentation in `docs/output.md` is updated.
- [X] `CHANGELOG.md` is updated.
- [X] `README.md` is updated (including new tool citations and authors/contributors).

## PR description

Added reviewer suggestion for version 2.2.0 release (#122)

### CODE REVIEW

File: .github/CONTRIBUTING.md [hunk 1]
```diff
@@ -26,11 +26,7 @@ If you're not used to this workflow with git, you can start with some [docs from
 
 ## Tests
 
-You have the option to test your changes locally by running the pipeline. For receiving warnings about process selectors and other `debug` information, it is recommended to use the debug profile. Execute all the tests with the following command:
-
-```bash
-nf-test test --profile debug,test,docker --verbose
-```
+You have the option to test your changes locally by running the pipeline.
```
Comment (d4straub) [inline .github/CONTRIBUTING.md:29]:
```suggestion
You have the option to test your changes locally by running the pipeline. For receiving warnings about process selectors and other `debug` information, it is recommended to use the debug profile.
```
Afaik this part is valid?

**Review by d4straub — COMMENTED** [review]
Just a comment

File: .github/CONTRIBUTING.md (hunk 1 above)
Comment (Daniel-VM) [reply .github/CONTRIBUTING.md]: Yess!! I added it again. Thanks ;).

**Review by d4straub — APPROVED** [review]
LGTM!

**Pull request merged**